apps/examples/smartfs/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+# This file is autogenerated, do not edit.
```

Comment:
As this line says, this is auto-generated file. This looks strange. Let me check another way.